### PR TITLE
Fix ms output

### PR DESCRIPTION
**Pull request by audhalbritter**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -36,7 +36,7 @@ data/
 *.jpeg
 *.jpg
 *.md
-*2.bib
+!Rmd/TDT2.bib
```
Comment (richardjtelford) [inline .gitignore:39]: why? TDT2.bib should not be edited or committed. It is generated with code from TDT.bib

File: Rmd/TDT2.bib [hunk 2]
```diff
@@ -0,0 +1,70 @@
+% Encoding: UTF-8
```
Comment (richardjtelford) [inline Rmd/TDT2.bib:1]: don't commit this file

File: .gitignore (hunk 1 above)
Comment (audhalbritter) [reply .gitignore:39]: I added a name in the traitsstrap packaged, I thought this was done manually...
how do I remove this file from the commit list then?

Comment (richardjtelford) [reply .gitignore:39]: line 40 of 1_trait_driver_drake.R has a list of packages to cite. Ensures that the correct version is cited.

**Pull request merged**